docs/intro/installation.rst

```diff
@@ -4,84 +4,37 @@
 Installation
 ************
 
-The SunPy project `maintains a range of packages <https://sunpy.org/project/affiliated>`__ that leverage the wider ecosystem of scientific Python packages for solar physics.
+This is the first chapter in the sunpy tutorial, and by the end of it you should have a working installation of Python and sunpy.
 
 Installing Python
 =================
-These instructions will set you up with `miniforge <https://conda-forge.org/docs/user/introduction.html>`__, which makes it easy to install and manage Python packages.
+There are many ways to install Python, but even if you have Python installed somewhere on your computer we recommend following these instructions anyway.
+That's because we will create a new Python environment.
+As well as containing a Python installation, this environment provides an isolated place to install Python packages (like sunpy).
 
-To install the miniforge Python distribution follow `the miniforge installation instructions <https://github.com/conda-forge/miniforge#install>`__.
+The package manager we'll be using is called ``conda``.
```

Comment:
I think this is the point where we should link to the explanation of different install options?

Reply:
I don't want to interrupt the flow of the tutorial, but have added a link at the end.

Reply:
This feels like a style decision we will have to make as we add more explanation style content. I think we are going to have this crop up in multiple places, i.e "do thing like this <here's why>".

Figuring out what a good approach(s) is might fall to you as the first person to make an explanation page :rofl: